webpack/scenes/ContentViews/Details/Versions/Delete/affectedActivationKeys.js

```diff
@@ -9,14 +10,16 @@ import {
   selectCVActivationKeys, selectCVActivationKeysStatus,
 } from '../../ContentViewDetailSelectors';
 import EnvironmentLabels from '../../../components/EnvironmentLabels';
-import DeleteContext from './DeleteContext';
 
-const AffectedActivationKeys = () => {
-  const { versionEnvironments, selectedEnvSet, cvId } = useContext(DeleteContext);
+const AffectedActivationKeys = ({
+  versionEnvironments, selectedEnvSet, cvId, deleteCV,
+}) => {
   const [searchQuery, updateSearchQuery] = useState('');
   const response = useSelector(state => selectCVActivationKeys(state), shallowEqual);
   const status = useSelector(state => selectCVActivationKeysStatus(state), shallowEqual);
-  const selectedEnv = versionEnvironments.filter(env => selectedEnvSet.has(env.id));
+  const selectedEnv = deleteCV ?
+    versionEnvironments :
+    versionEnvironments.filter(env => selectedEnvSet?.has(env.id));
 
   const fetchItems = useCallback(() => {
     const formSearch = () => {
```

Comment:
Why not: 
     cvQuery += ` AND (${selectedEnvStrings} )`;

     also, is there supposed to be a space after the strings?